dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -173,311 +173,347 @@ Resources:
+                "stacked": false,
+                "region": "${__Region__}",
+                "stat": "Sum",
+                "period": 60,
+                "title": "Delivery - issues"
+              }
+            },
+            {
+              "height": 6,
+              "width": 12,
+              "y": 42,
+              "x": 12,
+              "type": "metric",
+              "properties": {
+                "metrics": [
+                [ "DT/LogsStreaming", "Log attr trimmed", "function_name", "${__LambdaName__}" ],
+                [ ".", "Log content trimmed", ".", "." ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "region": "${__Region__}",
+                "period": 60,
+                "title": "Logs - trimmed",
+                "stat": "Sum"
+              }
+            },
+            {
+              "height": 6,
+              "width": 12,
+              "y": 6,
+              "x": 12,
+              "type": "metric",
+              "properties": {
+                "metrics": [
+                [ "DT/LogsStreaming", "Log age min", "function_name", "${__LambdaName__}", { "stat": "Minimum" } ],
+                [ "DT/LogsStreaming", "Log age avg", "function_name", "${__LambdaName__}" ],
+                [ "DT/LogsStreaming", "Log age max", "function_name", "${__LambdaName__}", { "stat": "Maximum" } ]
+                ],
+                "view": "timeSeries",
+                "stacked": false,
+                "title": "Logs - age",
+                "region": "${__Region__}",
+                "period": 60,
+                "stat": "Average"
+              }
+            },
+            {
+              "height": 6,
+              "width": 12,
+              "y": 0,
+              "x": 0,
+              "type": "metric",
+              "properties": {
+                "view": "timeSeries",
+                "stacked": false,
+                "metrics": [
+                [ "AWS/Lambda", "Invocations", "FunctionName", "${__LambdaName__}" ]
+                ],
+                "region": "${__Region__}",
+                "title": "Lambda - invocations"
+              }
+            },
+            {
+              "height": 6,
+              "width": 12,
+              "y": 0,
+              "x": 12,
+              "type": "metric",
+              "properties": {
+                "view": "timeSeries",
+                "stacked": false,
+                "metrics": [
+                [ "AWS/Lambda", "Duration", "FunctionName", "${__LambdaName__}" ]
+                ],
+                "region": "${__Region__}",
+                "title": "Lambda - duration"
+              }
+            },
+            {
+              "height": 6,
+              "width": 12,
+              "y": 12,
+              "x": 12,
+              "type": "metric",
+              "properties": {
+                "metrics": [
+                [ "...", "${__LambdaName__}" ]
```

Comment:
How is namespace and metric name determined in this case?

Reply:
Thanks, fixing, it seems I used to have there 2 same metrics for different functions, and ate up the part that actually defined those:

"metrics": [
                    [ "AWS/Lambda", "Errors", "FunctionName", "BeluAWSLogsPerformanceTest_inputLogsProducingLambda", { "visible": false } ],
                    [ "...", "BeluPerfLogsCLient-Lambda-1VXEVIDV8B4TC" ]
                ],